src/main/java/edu/cornell/kfs/sys/util/LoadFileUtils.java

```diff
@@ -44,4 +47,31 @@ public static byte[] safelyLoadFileBytes(File file) {
         }
         return safelyLoadFileBytes(file.getAbsolutePath());
     }
+    
+    /**
+     * Returns the string contents of a file.  If the file can not be loaded, an empty string is returned.
+     * Uses a default UTF_*
```

Comment:
Did you mean to say "UTF_8" in this comment?

Reply:
yes, will fix